lib/tasks/demo-build.js

```diff
@@ -339,7 +339,7 @@ module.exports = function (cfg) {
 	const config = cfg || {};
 	const cwd = config.cwd || process.cwd();
 	if (Boolean(config.demoConfig) && config.demoConfig !== 'origami.json') {
-		return Promise.reject(new Error('Custom demo config files are not supported, please place demo config inside of origami.json.'));
+		throw new Error('Custom demo config files are not supported, please place demo config inside of origami.json.');
```

Comment:
This should be a Promise rejection and not a throw because it is not inside a Promise chain and is actually being thrown and not casted into a Promise rejection.
```suggestion
		return Promise.reject(new Error('Custom demo config files are not supported, please place demo config inside of origami.json.'));
```